examples/lighting-app/linux/BUILD.gn

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("${chip_root}/gn/chip/tools.gni")
+
+assert(chip_build_tools)
+
+executable("chip-standalone-server") {
+  sources = [
+    "${chip_root}/src/app/clusters/on-off-server/on-off.c",
+    "${chip_root}/src/app/util/attribute-size.c",
+    "${chip_root}/src/app/util/attribute-storage.c",
+    "${chip_root}/src/app/util/attribute-table.c",
+    "${chip_root}/src/app/util/chip-response.cpp",
+    "${chip_root}/src/app/util/client-api.c",
+    "${chip_root}/src/app/util/ember-print.cpp",
+    "${chip_root}/src/app/util/message.c",
+    "${chip_root}/src/app/util/process-cluster-message.c",
+    "${chip_root}/src/app/util/process-global-message.c",
+    "${chip_root}/src/app/util/util.c",
+    "gen/call-command-handler.c",
+    "gen/callback-stub.c",
+    "gen/znet-bookkeeping.c",
+    "main.cpp",
+  ]
+
+  include_dirs = [
+    ".",
+    "${chip_root}/src/app/util",
```

Comment:
would prefer that we used `#include <app/util/XXX.h` in source files and and have `-I${chip_root}/src` if possible.  if there's an issue with a file in `gen/` that makes this pattern untenable, lemme know

Reply:
There are a bunch of includes in gen files of `af.h`, `af-types.h`, `util.h`.  More importantly, there are includes like that in the non-gen files too.  We could fix that by adding `${chip_root}/src` to the include path for all the apps involved, then changing the non-gen files, then changing the various gen files in the apps, then removing `${chip_root}/src/app/util` from the include paths...

I would argue for not trying to do that as part of this PR and doing a followup instead.